src/components/Post/newPost.js

```diff
@@ -210,55 +229,130 @@ const PostUpload = ({ isOpen, onClose }) => {
                                 leave="ease-in duration-200"
                                 leaveFrom="opacity-100 translate-y-0 sm:scale-100"
                                 leaveTo="opacity-0 translate-y-4 sm:translate-y-0 sm:scale-95">
-                                <Dialog.Panel className="max-height= relative transform overflow-hidden rounded-lg bg-white text-left shadow-xl transition-all sm:my-8 sm:w-full sm:max-w-lg">
-                                    <div className="bg-white px-4 pb-4 pt-5 sm:p-6 sm:pb-4">
-                                        <div className="sm:flex sm:items-start">
-                                            <div className="mt-3 text-center sm:ml-4 sm:mt-0 sm:text-left">
-                                                <Dialog.Title
-                                                    as="h3"
-                                                    className="text-base font-semibold leading-6 text-gray-900 pb-6">
-                                                    Post something.
-                                                </Dialog.Title>
-
-                                                <Dialog.Description>
-                                                    <div className="mb-4 required">
-                                                        <label
-                                                            htmlFor="title"
-                                                            className="flex justify-start block mb-1 text-sm font-medium text-gray-900 dark:text-black">
-                                                            Title
-                                                        </label>
-                                                        <input
-                                                            type="text"
-                                                            id="title"
-                                                            onChange={
-                                                                handleTitleChange
-                                                            }
-                                                            value={title}
-                                                            className="bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500"
-                                                            placeholder="Title..."
-                                                            required={true}
-                                                        />
-                                                    </div>
+                                <Dialog.Panel className="relative transform overflow-hidden rounded-lg bg-white text-left shadow-xl transition-all sm:my-8 sm:w-full sm:max-w-lg">
```

Comment:
Few things I'd like to address here:
- As discussed, we need to make it a full page. Not as a Dialog Panel. Let's discuss this in details tomorrow.
- The marker at the end ( | ) in title section is covering some part of last character. Please look if it can be fixed.
- Max hashtags allowed should be 3.
- Tags should be contained in a container and every tag should be separated with a space. Like the normal tag behaviour we're used to.
- While the media is being uploaded, the app feels dead. Check if we can add a progress bar or a spinner to keep users updated that the media is currently being uploaded.
- The post button should be disabled until Title and media both aren't selected. Tags can be optional for now. 
- There should be a close/Cancel button to allow users to get back to the scrolling flow.
- Once a picture is uploaded, we should have a change/remove button allowing users to change the uploaded media or remove it. Right now, the upload media button works fine for replacing though but not all users would consume it like that.